tracer/test/snapshots/AwsLambdaTests.verified.txt

```diff
@@ -150,13 +180,43 @@ at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, Cancellat
     }
   },
   {
-    TraceId: Id_1,
-    SpanId: Id_8,
+    TraceId: Id_7,
```

Comment:
This snapshot doesn't look right, should the http request to the function be failing? I expected all of the spans to be successful

Reply:
no they all fail since `http://localhost/function/xxx` are not a real endpoints.
It's just to check whether or not the spans have been generated (and then that the integration is working)
there is not point of actually creating a fake server in the container to answer 200 to those requests, since we just want to see a span
does it make sense?

Reply:
Oh I see, that makes sense. I didn't think we were expecting errors in the snapshots, but if I unfold the diff and look at unaffected areas of this file it is very clear that this has been the testing strategy all along. Thank you for clarifying that for me!👍🏼

Reply:
> I didn't think we were expecting errors in the snapshots

Likewise, I think that has confused everyone on the team by now 😆 

> there is not point of actually creating a fake server in the container to answer 200 to those requests

Actually, I think it would be pretty valuable, given we all keep getting confused by this 🙂 [in other samples](https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/test/test-applications/integrations/Samples.Telemetry/Program.cs) we have the sample app make requests to _itself_ which is relatively easy to do, and would mean we don't have any errors? How do you feel about that (could definitely be in a separate PR)